src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +608,13 @@
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		if(Skript.classExists("org.bukkit.event.player.PlayerAdvancementDoneEvent")) {
+			Skript.registerEvent("Advancement done", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "[on] advancement (done|finish)")
```

Comment:
```suggestion
			Skript.registerEvent("Advancement Completion", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "advancement (completion|finish)")
```
I think `completion` sounds better than `done` (I know it's the name of the event - but it's not a very good name imo)
Up for discussion :D

Reply:
> I think `completion` sounds better than `done` (I know it's the name of the event - but it's not a very good name imo)
> Up for discussion :D
> You also do not need "on" as skript will handle that

I agree about this, but should we add completion and done, since its the name of the event?

Reply:
Yeah, it wouldn't hurt to have it - just for clarity for users familiar with that event I suppose. I have amended my suggestion.